runner.rb

```diff
@@ -0,0 +1,15 @@
+require './lib/stat_tracker'
+
+game_path = './data/games.csv'
+team_path = './data/teams.csv'
+game_teams_path = './data/game_teams.csv'
+
+locations = {
+  games: game_path,
+  teams: team_path,
+  game_teams: game_teams_path
+}
+
+stat_tracker = StatTracker.from_csv(locations)
+
+require 'pry'; binding.pry
```

Comment:
Need to remove this PRY

Reply:
I removed the pry, and pushed it back to aps_runner_branch